troposphere/static/js/components/modals/project/ProjectInstanceLaunchModal.react.js

```diff
@@ -46,7 +46,7 @@ define(
         // MachineStore.get(providerId, identityId, machineId) called by versions/MachineList
```

Comment:
getState method here should also probably move AWAY from the 'getState' approach and move the 'heavy lifting' code into render (as suggested by @prosif and @steve-gregory)